lua/wire/wireshared.lua

```diff
@@ -41,6 +41,25 @@ function table.Compact(tbl, cb, n) -- luacheck: ignore
 	end
 end
 
+-- Works like #tbl, but works when item is removed from middle of the table without shifting.
+--[[
+	local t = {1,2,3,4,5}
+	t[3] = nil
+
+	print(#t) -- May return 5, as it is undefined behaviour
+	print(table.SeqCount(t)) -- Will return 2
+
+]]
+function table.SeqCount(tbl)
```

Comment:
Why not just use `table.remove`?

At least, you could use a variable to note the lowest value instead of counting like this.

```lua
lowest = 0

function insert(v)
    if lowest ~= 0 then
        tbl[lowest] = v
    else
        table.insert(tbl, v)
    end
end

function remove(k)
    tbl[k] = nil
    if k < lowest then lowest = k end
end